### ISSUE TITLE
[Feat]: static builds via kickstart are not predictable

### ISSUE THREAD

**Issue by ktsaou** [netdata/netdata#12255]
### Problem

Today, when you install netdata via the kickstart script, you may end up with an installation in `/opt/netdata`. This means that all configuration has now to be done in `/opt/netdata/etc` and all netdata commands need to be typed with `/opt/netdata/bin` in them.

### Description

Currently:

1. Users with a provisioning system can't know beforehand which version will be installed, so they don't know where to install custom configurations for netdata, or which paths to use for accessing netdata command lines.

2. Our documentation becomes too complex, having to explain all the time that depending on the installation users have, they may have to use different commands.

### Importance

must have

### Value proposition

1. Users should always be able to run netdata commands using the current path
2. Users should always be able to install configuration files in predictable paths (i.e. `/etc/netdata`)

### Proposed implementation

The static install should:

1. Symlink `/opt/netdata/etc` to `/etc/netdata`. If `/etc/netdata` already exists, files should be moved to `/opt/etc/netdata` and then do the symlink.
2. 3 new files should be created in `/usr/local/bin`:
   * `netdata`, symlink to `/opt/netdata/bin/netdata`
   * `netdatacli`, symlink to `/opt/netdata/bin/netdatacli`
   * `netdata-claim.sh` as a script like this:

```sh
#!/usr/bin/env sh

# ensure we use the statically built curl command
export PATH="/opt/netdata/bin:${PATH}"

# call the original script
/opt/netdata/bin/netdata-claim.sh "${@}"
```

The last point (3), could be just a symlink, if the `netdata-claim.sh` script sets properly the path by itself when it is installed with a statically linked netdata.

**Comment by Ferroin** [comment]
This would not actually eliminate the need to have install specific instructions for paths, just make it less likely that the user has to deal with a non-default case in some cases.

That said, I do think this is (generally) a good idea, though we need to figure out how we want to handle the case of there already being executables with the requisite names in whichever location we put things (also, should be `/usr/local/sbin`, not `/usr/local/bin`, for consistency with where we put these commands in other install methods).

**Comment by ilyam8** [comment]
> should be /usr/local/sbin, not /usr/local/bin

Then regular users (not root) won't be able to find/execute them.

**Comment by Ferroin** [comment]
> > should be /usr/local/sbin, not /usr/local/bin
> 
> Then regular users (not root) won't be able to find/execute them.

Regular users shouldn’t be trying to execute them. The claiming script does not work as a non-root user, and starting netdata as the ‘regular’ user it runs as actually does not work in many cases either.

That said, _most_ Linux distros actually include the `sbin` paths in their default search paths for regular users. The only one I know of which does not by default is Gentoo.

**Comment by ilyam8** [comment]
Sounds convincing, I agree.

**Comment by ktsaou** [comment]
You are right @Ferroin. I updated the ticket description to put them in `/usr/local/sbin`. Please proceed.
